Mlem/App/Views/Shared/ExpandedPost/ExpandedPostView.swift

```diff
@@ -198,6 +166,53 @@ struct ExpandedPostView: View {
         .environment(tracker)
     }
     
+    @ViewBuilder
+    func commentTree(tracker: CommentTreeTracker) -> some View {
+        ForEach(tracker.comments.itemTree(), id: \.hashValue) { item in
+            Group {
+                switch item {
+                case let .comment(comment):
+                    CommentView(
+                        comment: comment,
+                        highlight: [scrollTargetedComment?.actorId, highlightedComment?.actorId].contains(comment.actorId),
+                        depthOffset: tracker.proposedDepthOffset
+                    )
+                    .transition(.move(edge: .top).combined(with: .opacity))
+                    .zIndex(1000 - Double(comment.depth))
+                    .anchorPreference(
+                        key: AnchorsKey.self,
+                        value: .center
+                    ) { [comment.actorId: $0] }
+                    .padding(.leading, CGFloat(comment.depth - tracker.proposedDepthOffset) * 10)
+                    .id(comment.actorId)
+                case let .unloadedComments(comment, _):
+                    Button {
+                        navigation.push(.comment(comment))
+                    } label: {
+                        HStack {
+                            CommentBarView(depth: comment.depth + 1)
+                            HStack {
+                                Text("More Replies")
+                                Image(systemName: Icons.forward)
+                            }
+                            .frame(maxWidth: .infinity)
+                            .padding(.vertical, 2)
+                            .foregroundStyle(.secondary)
```

Comment:
*nit*

Or `.accent` (see general comment)

```suggestion
                            .foregroundStyle(palette.secondary)
```